packages/paste-core/components/table/src/types.ts

```diff
@@ -43,29 +53,29 @@ export const TrPropTypes = {
   verticalAlign: PropTypes.oneOf(Object.values(TableVerticalAlignmentObject)),
 };
 
-export interface ThProps {
-  children: NonNullable<React.ReactNode>;
+export interface ThProps extends React.ThHTMLAttributes<HTMLTableHeaderCellElement> {
+  children?: React.ReactNode;
   textAlign?: TableAlignmentOptions;
   width?: WidthOptions;
 }
 
 export const ThPropTypes = {
-  children: PropTypes.node.isRequired,
+  children: PropTypes.node,
   textAlign: PropTypes.oneOf(Object.values(TableAlignmentObject)),
   width: isWidthTokenProp,
 };
 
-export interface TdProps {
-  children: NonNullable<React.ReactNode>;
+export interface TdProps extends React.TdHTMLAttributes<HTMLTableCellElement> {
+  children?: React.ReactNode;
```

Comment:
ReactNode already handles undefined, so you can omit the `?`

Reply:
Interesting. If I remove the `?`, TS still thinks it's required.
https://share.getcloudapp.com/Wnuln1xA

Reply:
I could just remove `children` from the interface.

Reply:
Leaving as is for now.